pkgs/os-specific/darwin/iproute2mac/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, python }:
+
+stdenv.mkDerivation rec {
+  version = "1.2.1";
+  name = "iproute2mac-${version}";
+
+  src = fetchFromGitHub {
+    owner = "brona";
+    repo = "iproute2mac";
+    rev = "v${version}";
+    sha256 = "1n6la7blbxza2m79cpnywsavhzsdv4gzdxrkly4dppyidjg6jy1h";
+  };
+
+  buildInputs = [ python ];
+
+  postPatch = ''
+    substituteInPlace src/ip.py --replace /usr/bin/python ${python}/bin/python
```

Comment:
Can you also wrap the other commands here? darwin.network_cmds should have them all.

Reply:
right. what about sudo?

Reply:
It's probably okay to leave for now. I don't think it's packaged anywhere? /cc @LnL7

Reply:
patched all network_cmds. I doubt `pkgs/tools/security/sudo` would easily be portable for darwin, so relying on `/usr/bin/sudo` being there might be ok

Reply:
Sudo can't work, it needs a setuid bit.